frontend/src/routes/OpenChannel.tsx

```diff
@@ -58,7 +74,22 @@ export default function OpenChannel() {
 					<div />
 					<p className="text-2xl font-light">Let's do this!</p>
 					<div className="flex flex-col gap-4">
-						<input onChange={(e) => setPeerPubkey(e.target.value)} className={`w-full ${inputStyle({ accent: "blue" })}`} type="text" placeholder='Target node pubkey' />
+						{peers && peers.length &&
+							<div className="flex flex-col gap-2">
+								<label className="text-xl font-light">Pick the peer</label>
+								<div className="select-wrapper">
+									<select onChange={handleSelectChange} className={selectStyle({ accent: "blue", overflow: "yes" })} value={peerPubkey} placeholder="Network">
+										{peers.map(p => <option key={p.pubkey} value={p.pubkey}>{p.pubkey}</option>)}
+									</select>
+								</div>
+							</div>
+						}
+						{(!peers || !peers.length) &&
+							<div className="flex flex-col gap-2">
+								<label>New Peer</label>
+								<input onChange={(e) => setPeerPubkey(e.target.value)} value={peerPubkey} className={`w-full ${inputStyle({ accent: "blue" })}`} type="text" placeholder='Target node pubkey' />
+							</div>
+						}
```

Comment:
I think this won't work because we currently don't allow opening a channel to a peer we aren't connected to

Reply:
oh TIL